### PR TITLE
Cleanup build

### PR DESCRIPTION
**Pull request by stefanpenner**
- [x] some module paths are slightly wrong (Breaking tests)
- [x] ensure ember can consume this.
- [ ] sourceMap aware prepend of license. https://github.com/stefanpenner/broccoli-stew/issues/16

### CODE REVIEW

File: tests/cancel-test.js [hunk 1]
```diff
@@ -1,17 +1,17 @@
-import Backburner from "backburner";
+import Backburner from 'backburner';
```
Comment (seanpdoyle) [inline tests/cancel-test.js]: @stefanpenner should these single quotes be included in this PR? Could they be in a subsequent PR?

They're noisy and making it hard to tell where your main changes are.

**Comment by seanpdoyle** [comment]
@stefanpenner could you rebase this?

**Comment by stefanpenner** [comment]
@seanpdoyle done

**Comment by seanpdoyle** [comment]
:clap:

**Comment by stefanpenner** [comment]
> ensure ember can consume this.

got time for this one?

> sourceMap aware prepend of license. stefanpenner/broccoli-stew#16

we can likely punt on this for now, solve it later and circle back

**Comment by seanpdoyle** [comment]
@stefanpenner what would the process for that entail?

point the npm dependency on emberjs to this branch?

**Comment by stefanpenner** [comment]
> @stefanpenner what would the process for that entail?
> 
> point the npm dependency on emberjs to this branch?

a

**Comment by seanpdoyle** [comment]
@stefanpenner tracked here: https://github.com/emberjs/ember.js/pull/10853

**Comment by seanpdoyle** [comment]
@stefanpenner yup, failed: https://travis-ci.org/emberjs/ember.js/jobs/57730109#L150-L161

**Comment by stefanpenner** [comment]
Looks good. Now to update the rest of the micro libs

**Comment by seanpdoyle** [comment]
@stefanpenner I'm not sure what you mean. 

How does it look good?

What are the next steps?

File: lib/backburner.js [hunk 2]
```diff
@@ -6,20 +6,17 @@ import {
...
-} from 'backburner/utils';
+} from './backburner/utils';
 
-import {
+import platform, {
    needsIETryCatchFix
-} from 'backburner/platform';
-
-import searchTimer from 'backburner/binary-search';
+} from './backburner/platform';
```
Comment (seanpdoyle) [inline lib/backburner.js]: @stefanpenner is this valid syntax?

```js
import platform, {
  needsIETryCatchFix
} from './backburner/platform';
```

**Comment by stefanpenner** [comment]
No accidental comma at the end

**Comment by seanpdoyle** [comment]
@stefanpenner I don't think we're exporting modules properly

https://travis-ci.org/emberjs/ember.js/jobs/57733845#L157

File: lib/backburner.js (hunk 2 above)
Comment (seanpdoyle) [inline lib/backburner.js]: @stefanpenner corrected in https://github.com/ebryn/backburner.js/pull/134

**Comment by rwjblue** [comment]
Can you try updating your Ember pull-request to use https://github.com/emberjs/emberjs-build/pull/84 along with this branch?

**Comment by seanpdoyle** [comment]
@rwjblue :boom: https://travis-ci.org/emberjs/ember.js/jobs/57735935#L164-L177

**Comment by rwjblue** [comment]
Yes, that is why the emberjs-build changes haven't landed. More details in https://github.com/ebryn/backburner.js/issues/128.

**Pull request merged**